### PR TITLE
finished documentation for openshift

### PR DESCRIPTION
**Pull request by rimolive**
Adding instructions about how to use Rocket.Chat on OpenShift Online. Please send me feedback to improve the docs.

### CODE REVIEW

File: 3. Installation/2. PaaS Deployments/OpenShift/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Deployinng Rocket.Chat on OpenShift
```
Comment (geekgonecrazy) [inline 3. Installation/2. PaaS Deployments/OpenShift/README.md]: One slight tweak here.  Deployinng->Deploying 👍

File: 3. Installation/2. PaaS Deployments/OpenShift/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+4. [Create the app](#4-create-the-app)
+
+### 1. Login
+
+First of all we need to login to OpenShift API URL to create our project. The command to login to
+OpenShift Online Next Gen is shown below: 
+```bash
+# oc login https://api.preview.openshift.com --token=<api_token>
+```
+You can findthe API Token to login to OpenShift Online Next Gen in the about page of the Web console.
```
Comment (geekgonecrazy) [inline 3. Installation/2. PaaS Deployments/OpenShift/README.md]: findthe -> find the

**Comment by geekgonecrazy** [comment]
Thanks for the contribution!  👍  Added a couple of notes in line.  If you could make those few changes, it'd be greatly appreciated!  👍

**Comment by rimolive** [comment]
I fixed the typos you pointed out. Thanks

**Comment by geekgonecrazy** [comment]
@rimolive Much appreciated!  Thanks again for your contribution!  👍

**Pull request merged**